### PR TITLE
Add hero banner component

### PR DESCRIPTION
**Pull request by GusBaamonde**
**Add hero banner component**

hero-banner-component

- general adjustments to titles and paragraphs
- add atomic classes to text align

**you can see it:**
http://cdn.fromdoppler.com/doppler-ui-library/build816/hero-banner-component.html

#Checklist

- [X] Check if there is a new color.
- [X] Create the color variable in the _colors.scss file.
- [X] Follow the color nomenclature.
- [X] Create the color class with its variable.
- [X] Create the color component in the html, show the name, color class and its exadecimal.

### CODE REVIEW

File: assets/scss/core/_typography.scss [hunk 1]
```diff
@@ -85,41 +85,52 @@ h5,
...
+  padding-bottom: $dp-spaces--lv4;
+  font-weight: $dp-font-weight-bold;
 }
 
 h2 {
-  margin: 0;
   font-size: $dp-sizing--lvl6;
-  line-height: 28px;
-  font-family: $dp-font-family;
+  line-height: $dp-spaces--lv5;
```
Comment (fcoronelmakingsense) [inline assets/scss/core/_typography.scss:101]: this changes in the titles tags affect all the webapp?

File: assets/scss/core/_variables.scss [hunk 2]
```diff
@@ -83,3 +83,15 @@ $sides: (top, bottom, left, right); /// Directions
     }
   }
 }
+
+.text-align-r {
```
Comment (cbernat) [inline assets/scss/core/_variables.scss:87]: How about using BEM here? `text-align--right` The same with the other classes, in this file.

File: assets/scss/templates/_reportes.scss [hunk 3]
```diff
@@ -44,39 +44,26 @@
   background: $dp-color-ghostwhite;
 }
 
+/* TO DO: The .report-filter class will deprecate in the next version because it is not a generic class for this component.
+It will be replaced by .hero-banner*/
```
Comment (cbernat) [inline assets/scss/templates/_reportes.scss:48]: ❤️

File: assets/scss/core/_typography.scss (hunk 1 above)
Comment (GusBaamonde) [reply assets/scss/core/_typography.scss:101]: It probably affects in a general way, that's why I have to check the webapp well before merging

**Comment by GusBaamonde** [comment]
test this please

**Comment by mdirago** [comment]
Great Gus!

We've decided to not include the legend phrase in this case
https://prnt.sc/s176zc

In some cases, the 'with link' scenario should look like this. And the button can be enabled or not.
Captura de Pantalla 2020-04-17 a la(s) 10 20 24

**Comment by GusBaamonde** [comment]
test this please

**Pull request merged**